src/aktualizr_secondary/update_agent_ostree.cc

```diff
@@ -0,0 +1,91 @@
+#include "update_agent_ostree.h"
+
+#include "package_manager/ostreemanager.h"
+
+// TBD: consider moving this and SotaUptaneClient::secondaryTreehubCredentials() to encapsulate them in one place that
```

Comment:
Any reason to chose "TBD" over "TODO"? It makes grepping more challenging, unless that was the goal.

Reply:
will replace it with TODO.